Pomodoro/app/src/main/java/no/uia/ikt205/pomodoro/MainActivity.kt

```diff
@@ -1,52 +1,90 @@
 package no.uia.ikt205.pomodoro
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.os.CountDownTimer
+import android.view.Gravity
 import android.view.View
-import android.widget.Button
-import android.widget.TextView
-import android.widget.Toast
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
 import no.uia.ikt205.pomodoro.util.millisecondsToDescriptiveTime
 
-class MainActivity : AppCompatActivity() {
+
+class MainActivity : AppCompatActivity(), AdapterView.OnItemSelectedListener {
 
     lateinit var timer:CountDownTimer
     lateinit var startButton:Button
     lateinit var coutdownDisplay:TextView
+    lateinit var dropDown:Spinner
```

Comment:
Jeg ser at du har lagt til en dropdown meny her i stedet for knapper. I oppgaven ble det etterspurt knapper, så det kan være en idé å endre dette :)

Reply:
Synes det er mye ryddigere med dropdown og derfor har jeg valgt denne løsningen :)